docs/source/examples/pt_zne.md

```diff
@@ -0,0 +1,400 @@
+
+## Noisy ZNE
+
+Let's define a larger circuit of CNOT and H gates. 
+
+```{code-cell} ipython3
+
+from mitiq.benchmarks import generate_ghz_circuit
+
+circuit = generate_ghz_circuit(n_qubits=7)
+
+print(circuit)
+
+```
+
+We are going to add coherent noise to this circuit and then get the error-mitigated expectation value. For a detailed discussion on this, refer to the [ZNE user guide](../guide/zne-1-intro.md). 
+
+As we are using a simulator, we have to make sure the noise model adds coherent noise to CZ/CNOT gates in our circuit. For this, `get_noise_model` is used to add noise to CZ/CNOT gates. See [PT user guide](../guide/pt-1-intro.md) for more. 
+
+```{code-cell} ipython3
+from numpy import pi
+from cirq import CircuitOperation, CXPowGate, CZPowGate, DensityMatrixSimulator
+from cirq.devices.noise_model import GateSubstitutionNoiseModel
+
+def get_noise_model(noise_level: float) -> GateSubstitutionNoiseModel:
+    """Substitute each CZ and CNOT gate in the circuit
+    with the gate itself followed by an Ry rotation on the output qubits.
+    """
+    rads = pi / 2 * noise_level
+    def noisy_c_gate(op):
+        if isinstance(op.gate, (CZPowGate, CXPowGate)):
+            return CircuitOperation(
+                Circuit(
+                    op.gate.on(*op.qubits), 
+                    Ry(rads=rads).on_each(op.qubits),
+                ).freeze())
+        return op
+
+    return GateSubstitutionNoiseModel(noisy_c_gate)
+
+def execute(circuit: Circuit, noise_level: float):
+    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit."""
+    return (
+        DensityMatrixSimulator(noise=get_noise_model(noise_level=noise_level))
+        .simulate(circuit)
+        .final_density_matrix[0, 0]
+        .real
+    )
+
+
+# Set the intensity of the noise
+NOISE_LEVEL = 0.2
+
+
+# Compute the expectation value of the |0><0| observable
+# in both the noiseless and the noisy setup
+ideal_value = execute(circuit, noise_level=0.0)
+noisy_value = execute(circuit, noise_level=NOISE_LEVEL)
+
+NUM_TWIRLED_VARIANTS = 300
+twirled_circuits = generate_pauli_twirl_variants(circuit, num_circuits=NUM_TWIRLED_VARIANTS)
+
+# Average results executed over twirled circuits
+from functools import partial
+from mitiq import Executor
+pt_vals = Executor(partial(execute, noise_level=NOISE_LEVEL)).evaluate(twirled_circuits)
+twirled_result = np.average(pt_vals)
+
+
+print(f"Error without twirling: {abs(ideal_value - noisy_value) :.3}")
+print(f"Error with twirling: {abs(ideal_value - twirled_result) :.3}")
+```
+
+```{caution}
+It is worth noting that Pauli twirling's goal is to tailor the noise from coherent to incoherent. Depending on the noise strength, type of coherent noise etc. this transformation might not give better results after the Pauli twirled circuit is executed. See the plot in the next section for more information.
```

Comment:
```suggestion
It is worth noting that Pauli twirling's goal is to tailor the noise from coherent to incoherent.
Depending on the noise strength, type of coherent noise etc., this transformation alone might not give better results after the Pauli twirled circuit is executed.
```

Reply:
>  See the plot in the next section for more information.

Why remove this line?

Reply:
I'm not sure I meant to, but can we change it to

> See the plot in the next section for an example.